docs/launcher.rst

```diff
@@ -15,7 +15,44 @@ In the following sections, we consider the
 :ref:`QuickStart` guide.
 
 ..
-    TODO: Add section on ForkLauncher, Slurm, SGE
+    TODO: Add section on Slurm, SGE
+
+Fork Launcher
+-------------
+
+Simexpal uses the fork launcher if no further configurations are made. The fork launcher launches
+experiments on the local machine. In contrast to the :ref:`QueueLauncher`, the fork launcher
+requires an active connection during the execution of the experiments.
```

Comment:
It is not clear here what an "active connection" is. Maybe state something like: immediately (in the current terminal) and blocks until completion of the experiments.

Reply:
Done. I changed it to ``In contrast to the :ref:`QueueLauncher`, the fork launcher launches the experiments immediately in the current terminal and blocks until completion of the experiments``